.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on:
+  pull_request:
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2.1.0
+        with:
+          fetch-depth: 100
+      - name: Fetch tags
+        run: git fetch --depth=100 origin +refs/tags/*:refs/tags/*
+      - uses: olafurpg/setup-scala@v7
+        with:
+          java-version: adopt@1.11
+      - uses: olafurpg/setup-gpg@v2
```

Comment:
Why is please this required for PR builds?

Reply:
because I wanted to keep the PR and master builds as similar as possible, for consistency sake, so that we can catch every and all issue already in PR (even corner cases related to CI setup, for example)

Reply:
I got the consistency point 👌 

But again - it is a one-man-show project even with invalid README 😢 I would not prefer to have _infrastructure_ dependencies on such projects.

Reply:
I understand your points. A project maintained by more people, with valid README and officially recommended in GitHub Actions documentation would be better. But we don't have anything like that, do we? As things are, this is the most popular way how to publish to Sonatype. (I don't even know of any other way how to do it.)
And it's good enough -- it's reliable and it works as other projects successfully using it prove.

(What's invalid about that README? Am I missing something? Not that this is the most important thing, ofc... :D )

Reply:
Looks like the README is just copy of some generic template.
![image](https://user-images.githubusercontent.com/273052/91447194-b1819300-e878-11ea-88f3-5d8d4a58f807.png)

Delegating signing to a library that even don't have a proper documentation looks like thing we shouldn't do 😢

Reply:
:facepalm: :smile: of course you're right... I was so focused on finding a malformed markdown or something like that, that I didn't see the forest for the trees